Examples/arcs-make

```diff
@@ -188,8 +191,8 @@ $(draft).tigmint.fa: $(draft).fa $(reads).fq.gz
 	$(gtime) tigmint tigmint draft=$(draft) reads=$(reads) minsize=$(minsize) as=$(as) nm=$(nm) dist=$(dist) mapq=$(mapq) trim=$(trim) span=$(span) window=$(window) t=$t 
 
 #Pre-processing long reads; cut into shorter segments (pseudo-linked reads)
-%.cut$(cut).fq.gz: $(long_reads)
-	$(gtime) sh -c '$(bin)../src/long-to-linked-pe -l $(cut) -t $t -m2000 $(long_reads) | $(gzip) > $@'
+$(reads).cut$(cut).fq.gz: $(long_reads)
+	$(gtime) sh -c '$(bin)/../src/long-to-linked-pe -l $(cut) -t $t -m2000 $(long_reads) | $(gzip) > $@'
 
```

Comment:
Is that slash needed

Reply:
It's just keeping the command consistent with the other `long-to-linked-pe` invocations